fedimint-server/src/atomic_broadcast/mod.rs

```diff
@@ -1,122 +0,0 @@
-//! This module implements fedimints custom atomic broadcast abstraction. A
-//! such, it is responsible for ordering serialized items in the form of byte
-//! vectors. The Broadcast is able to recover from a crash at any time via a
-//! backup that it maintains in the servers [fedimint_core::db::Database]. In
-//! Addition, it stores the history of accepted items in the form of
-//! signed session outcomes in the database as well in order to catch up fellow
-//! guardians which have been offline for a prolonged period of time.
-//!
-//! Though the broadcast depends on [fedimint_core] for [fedimint_core::PeerId],
-//! [fedimint_core::encoding::Encodable] and [fedimint_core::db::Database]
-//! it implements no consensus logic specific to Fedimint, to which we will
-//! refer as Fedimint Consensus going forward. To the broadcast a consensus item
-//! is merely a vector of bytes without any further structure.
-//!
-//! # The journey of a ConsensusItem
-//!
-//! Let us sketch the journey of an [fedimint_core::epoch::ConsensusItem] into a
-//! signed session outcome.
```

Comment:
That sounds like an interesting writeup, why remove it? Is it too out of date?

Reply:
It is mostly out of date now.